build.gradle.kts

```diff
@@ -275,6 +275,7 @@ tasks {
     prepareTestingSandbox {
         doLast {
             copyPlugins(destinationDir, pluginName)
+            renameCsharpPlugins(destinationDir, pluginName)
```

Comment:
Why did we need this?

Reply:
With the new C# repackaging, we rename the plugins in the /plugins/ directory so we can easily reference it in the code (with a static variable). This was done for the normal environment, but in a test environment I forgot to add this renaming, so I noticed some errors in some tests because the plugin couldn't be loaded. It didn't break the CI probably because we don't have a unit test for C#.